tauri-utils/src/platform.rs

```diff
@@ -51,3 +53,28 @@ pub fn target_triple() -> Result<String, crate::Error> {
 
   Ok(format!("{}-{}", arch, os))
 }
+
+pub fn resource_dir() -> Result<PathBuf, crate::Error> {
+  let exe = std::env::current_exe()?;
+  let exe_dir = exe.parent().expect("failed to get exe directory");
```

Comment:
these expects can be cleaned up with `?`s.  If they don't work then add the errors to the `error_chain` macro.

Reply:
I think I've tried to do that, but those are Options instead of Results.

Reply:
Alright, if they are options then the expects are fine.